sqlmesh/core/console.py

```diff
@@ -912,6 +912,9 @@ def _show_missing_dates(self, plan: Plan, default_catalog: t.Optional[str]) -> N
             backfill = self._limit_model_names(backfill, self.verbose)
         self._print(backfill)
 
+    def _validate_date_input(self, date_str: str) -> None:
```

Comment:
I don't see any validation happening here. Should this be `_print_date_time_input`?